webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -24,9 +24,11 @@ function GithubItems(props) {
         const repoName = item.repository_url ? item.repository_url.replace(/.+\/repos\//, '') : item.repository.full_name;
 
         let userName = null;
+        let userProfile = null;
 
         if (item.user) {
             userName = item.user.login;
+            userProfile = `https://github.com/${userName}`;
```

Comment:
What do you think about this:

```js
if (item.user) {
    userName = item.user.login;
} else if (item.owner) {
    userName = item.owner.login;
}

const userProfile = `https://github.com/${userName}`;
```

Reply:
Good catch!

Reply:
Actually `userName` may not be defined. So maybe just construct the URL inline since it's only used once:

```jsx
<a
    href={`https://github.com/${userName}`}
>
    {userName}
</a>
```